kinto_wizard/kinto2yaml.py

```diff
@@ -20,7 +20,7 @@ def _sorted_principals(permissions):
     """
     items = dct.items()
     results = await asyncio.gather(*(item[1] for item in items))
-    return dict(zip((item[0] for item in items), results))
+    return {k: v for k, v in zip((item[0] for item in items), results) if v is not None}
```

Comment:
I don't remember why I changed the behaviour here :confused:

Reply:
Apparently it fixes the case of empty buckets with the `introspect_server` call:

```
        with open("tests/dumps/dump-toto.yaml") as f:
>           assert f.read() == generated
E           AssertionError: assert 'natim:\n  co...a5494b127c4\n' == 'date: null\nn...a5494b127c4\n'
E             + date: null
E               natim:
```

Reply:
It could deserve some intermediary variables and/or comments :)

Reply:
```suggestion
    keys_results =  zip((item[0] for item in items), results)
    return {k: v for k, v in keys_results if v is not None}
```